### PR TITLE
Showing the Popover issue

### PR DESCRIPTION
**Pull request by alexandcote**
This PR replicates an issue we found in version >=7.4.0 of Polaris

### Screenshot

https://user-images.githubusercontent.com/4728325/153961648-31df08c8-1883-4e0a-bc21-4b2bb2b5d0f4.mov

[This PR](https://github.com/Shopify/polaris-react/pull/4580) introduce the functionality

cc @dfmcphee

### CODE REVIEW

File: playground/Playground.tsx [hunk 1]
```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 
-import {Page} from '../src';
-
 export function Playground() {
   return (
-    <Page title="Playground">
-      {/* Add the code you want to test in here */}
-    </Page>
+    <div
```
Comment (alexandcote) [inline playground/Playground.tsx:5]: This component only replicates the small window.

File: playground/PopoverFrame.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  );
+
+  const activator = (
+    <Button onClick={togglePopoverActive} disclosure>
+      More actions
+    </Button>
+  );
+
+  return (
+    // Frame without topBar doesn't have the issue.
```
Comment (emma-boardman) [inline playground/PopoverFrame.tsx:19]: I can still replicate the issue both in `Playground` in a `Frame` without the `TopBar`, and in online-store-web after manually removing the `data-polaris-top-bar` attribute from the `TopBar` div.

It seems to be related to whether the activator is on screen? Going to dig further. FWIW, the Popover is being closed by the `ScrollOut` event. 

https://user-images.githubusercontent.com/17032120/154038886-744640c4-5004-434f-8414-5bc435eed59e.mov

**Comment by emma-boardman** [comment]
Merchant videos for additional context:

- https://screenshot.click/22-02-0ftp5-i4rbe.mp4

- https://videobin.shopify.io/v/opepx8

**Comment by emma-boardman** [comment]
Closed in favour of: https://github.com/Shopify/polaris-react/pull/5181

**Pull request closed without merging**